api/app/signals/apps/api/tests/test_history_endpoint.py

```diff
@@ -26,7 +26,7 @@
 
 class TestHistoryAction(SIAReadWriteUserMixin, SignalsBaseApiTestCase):
     def setUp(self):
-        self.signal = SignalFactory.create()
+        self.signal = SignalFactory.create(user_assignment=None)
```

Comment:
Is this to make sure no user assignment is created to a user with the value None? Becuase with these changes it will appear in the history?

```
user_assignment = RelatedFactory('signals.apps.signals.factories.signal_user.SignalUserFactory', '_signal', user=None) # noqa
```

Reply:
This was exactly to make these assignments show up in the history for a signal. Previously we could not set a user, fully remove the user and then set it again. With this change that is possible (using the old-style history for now).